cleanup.sh

```diff
@@ -8,6 +8,17 @@ if [ -z "$TERMINUS_SITE" ] || [ -z "$TERMINUS_ENV" ]; then
 fi
 
 ###
+# Report the results to Slack (privately)
+###
+set +x
+if [ ! -z "$SLACK_WEBHOOK" ] && [ ! -z "$SLACK_CHANNEL" ]; then
+	PANTHEON_SSH_USER=$(terminus site connection-info --field=sftp_username)
+	PANTHEON_SSH_HOST=$(terminus site connection-info --field=sftp_host)
+	ssh -p 2222 -o StrictHostKeyChecking=no $PANTHEON_SSH_USER@$PANTHEON_SSH_HOST wp phpunit-report $SLACK_WEBHOOK $SLACK_CHANNEL $CIRCLE_BUILD_URL
```

Comment:
I'm not certain, @greg-1-anderson? Also, shouldn't this go in the `notify` step of CircleCI?